webapp/src/main/java/com/box/l10n/mojito/service/scheduledjob/ScheduledJobManager.java

```diff
@@ -210,7 +211,8 @@ private ScheduledThirdPartySyncProperties getScheduledThirdPartySyncProperties(
         new ScheduledThirdPartySyncProperties();
     thirdPartySyncProperties.setThirdPartyProjectId(jobConfig.getThirdPartyProjectId());
     thirdPartySyncProperties.setActions(jobConfig.getActions());
-    thirdPartySyncProperties.setPluralSeparator(jobConfig.getPluralSeparator());
+    thirdPartySyncProperties.setPluralSeparator(
+        Objects.equals(jobConfig.getPluralSeparator(), "") ? " _" : jobConfig.getPluralSeparator());
```

Comment:
Nit: it would be nice to have this as a constant called something like `DEFAULT_PLURAL_SEPARATOR`